abi-spec/utils.go

```diff
@@ -0,0 +1,25 @@
+package abispec
+
+import (
+	"fmt"
+	"math/big"
+)
+
+func HexToNumber(hex string) interface{} {
+	num, success := big.NewInt(0).SetString(hex, 16)
+	if success {
+		if num.Cmp(maxSafeInteger) == 1 {
```

Comment:
why not always return a string?

Reply:
just want to keep return same result as status-mobile did before, implementation for `hex-to-number` in status-mobile may return type of string or numeric @flexsurfer

Reply:
i think now it's better to return always string

Reply:
sure

> i think now it's better to return always string